pkg/ca/fileca/load.go

```diff
@@ -27,21 +27,34 @@ import (
 	"go.step.sm/crypto/pemutil"
 )
 
-func loadKeyPair(certPath, keyPath, keyPass string) (*x509.Certificate, crypto.Signer, error) {
+func loadKeyPair(certPath, keyPath, keyPass string) ([]*x509.Certificate, crypto.Signer, error) {
 
 	var (
-		cert *x509.Certificate
-		err  error
-		key  crypto.Signer
+		certs []*x509.Certificate
+		err   error
+		key   crypto.Signer
 	)
 
-	// TODO: Load chain of certs (intermediates and root) instead of just one
-	// certificate.
-	cert, err = pemutil.ReadCertificate(certPath)
+	certs, err = pemutil.ReadCertificateBundle(certPath)
```

Comment:
It's worth documenting the expected order of certificates. From ReadCertBundle, this would be intermediate first, root last.

Reply:
I've documented this in a comment in the code, but the entire `fileca` backend needs documentation as well. I've added a note to #302 to explicitly add this to the docs. Are you alright with catching the docs in that issue / in an iteration?

Reply:
Good with me! Thanks for adding the comment.